data/crac-creation/crac-creator-cse/src/main/java/com/farao_community/farao/data/crac_creation/creator/cse/critical_branch/CseCriticalBranchCreationContext.java

```diff
@@ -84,7 +84,7 @@ public boolean isSelected() {
         return selected;
     }
 
-    CseCriticalBranchCreationContext(CriticalBranchReader criticalBranchReader) {
+    public CseCriticalBranchCreationContext(CriticalBranchReader criticalBranchReader) {
```

Comment:
I would prefer you put the MonitoredElements classes in the same package, since you are already sharing some code between two features